app/assets/stylesheets/application.css.scss

```diff
@@ -91,16 +97,39 @@ body[data-action="devise/sessions-new"], body[data-action="devise/sessions-creat
   }
 
   body {
+    background-color: $bg-color;
     margin: 0 0 20px;
   }
 
-}
+  form {
+    background: $white;
+    border: 1px solid $border;
+    border-collapse: inherit; 
+    border-radius: 5px;
+    margin-bottom:20px; 
+    padding: 20px;
+  }
 
-// td.hover-actions {
-//   > a.action { display: none; }
-//   &:hover > a.action { display: inline; }
-// }
+  #login-box form {
+    border:none;
+    border-radius: none;
+    margin: 0;
+    padding: 0;   
+  }
 
+  table.users tr:nth-of-type(2) th {
+    border-top:0;
```

Comment:
Properties should be ordered border-color, border-top<br>Colon after property should be followed by one space